### DISCUSSION TITLE
Définitions des règles de transcription

### DISCUSSION THREAD

**Discussion by saranadal** [anrExperts/adire#9] [Q&A]
Ce que propose Ariane Pinche concerne les textes médiévaux. Doit-on suivre ses recommandations ?
Il y a celles aussi de Maxime Gohier sur les textes français du XVIIe

**Comment by sardinecan** [comment]
- Ariane Pinche : https://hal.science/hal-03697382/document
- Nouvelle France : https://lesgardenotes.org/kb/regles-de-transcription/
- Barbiche : http://theleme.enc.sorbonne.fr/cours/edition_epoque_moderne/edition_des_textes

**Comment by sardinecan** [comment]
voir aussi :
Le projet "Crimes et châtiments" a par exemple fait le choix de développer les abréviations puis de les mettre en notes dans Transkribus pour obtenir un corpus différent (cf. Élodie Paupe. ENC. (2022, 23 juin). 1.4 : Une cursive du 17e siècle , in Documents anciens et reconnaissance automatique des écritures manuscrites. [Vidéo]. Canal-U. [https://www.canal-u.tv/133333].)

**Comment by emchateau** [comment]
ll faut faire attention au fait que Transkribus a un comportement particulier.
À mon avis, nous avons intérêt à nous orienter vers une stratégie la plus orientée possible sur la tâche que l’on souhaite réaliser en écartant l’idée d’édition. Ce que l’on doit produire, ce sont des données d’entraînement. Nos critères devraient être la qualité des annotation produites.

**Comment by emchateau** [comment]
Salut @saranadal j'ai regardé en détail le guide d’Arianne, il va clairement falloir adapter pour la période moderne. Notamment pour les accentuations.
Peux-tu commencer un document qui récapitulerait les cas de figure, stp ?

**Comment by saranadal** [comment]
Bonjour @emchateau, je suis d'accord avec toi, il faut à mon avis s'écarter de l'idée éditoriale. Même s'il s'agit de textes médiévaux, Ariane Pinche ne perd pas de vue qu'il s'agit d'effectuer à terme des modèles d'HTR. La notion de bruit est primordiale. A première vue, j'ai l'impression que les règles proposées par Nouvelle France sont proches de ce que l'on recherche. Je m'occupe du document récapitulatif. J'ai partagé le tableau comparatif sur le drive avec vous trois.

**Comment by emchateau** [comment]
J'avais un peu participé à la discussion pour la Nouvelle France, mais je ne suis pas certain par exemple qu’ils ne développent pas les abréviations. 
Merci beaucoup pour le tableau. Attention j’ai deux compte sur Drive, j'utilise désormais l’adresse de l’Université pour le travail. Est-ce que le document est bien dans notre dossier experts ?

**Comment by sardinecan** [comment]
je l'ai replacé dans le dossier GIP 2022 => comparaisonNormesDeTranscription

**Comment by sardinecan** [comment]
Une lecture complémentaire : [le compte rendu de séance n°1 du CremmaLab](https://cremmalab.hypotheses.org/seminaire-creation-de-modeles-htr/compte-rendu-de-la-seance-n1)

**Comment by sardinecan** [comment]
Au regard de la bibliographie, j'ai l'impression que la généricité d'un modèle dépend du nombre d'intervention lors de la transcription : plus on force un modèle à interpréter, moins le modèle sera générique.

**Comment by sardinecan** [comment]
La stratégie admise (tout du moins par le CremmaLab) c'est de travailler en étapes spécialisées
- transcription la plus fidèle possible
- segmentation des mots (agglutination/séparation)
- développement des abréviations
- etc.
- éditorialisation

L'avantage de cette méthode étant de pouvoir entrainer des modèles ou d'adapter des stratégies à chaque étape.

**Comment by sardinecan** [comment]
Cela dit, ces étapes spécialisées demandent beaucoup de ressources (temps et humain). Il est donc primordial de bien définir nos besoins et de replacer le HTR dans notre chaine de traitement pour que son utilisation soit efficace.

**Comment by sardinecan** [comment]
J'aurais donc tendance à penser qu'il faut laisser de côté l'idée d'une éditorialisation, car ce n'est pas notre objectif premier. De plus, je ne pense pas que nous serons en mesure de tout résoudre lors de la transcription primaire (abréviations, ponctuation, agglutinations et séparations des mots, majuscules, etc.) sauf à y passer un temps que nous n'avons pas.

**Comment by sardinecan** [comment]
@emchateau @saranadal :
Comme base de discussion, je propose de suivre plus ou moins ce que propose Nouvelle France (qui ne semble pas très interventionniste) :
- non développement des abréviations 
- maintien du suscrits
- maintien de l'accentuation
- maintien des majuscules / minuscules
- séparation des mots agglutinés / réunion des mots séparés selon usage actuel
- maintien de l'apostrophe si présente, ne pas ajouter d'espace ou d'apostrophe si absente
- résolution i/j u/v selon usage actuel
- transcrire le ſ (s long) par s (différent de Nouvelle France, qui le maintient)

Reste les questions : 
- des ratures, faut-il les transcrire, quid lorsqu'elles sont illisibles ? 
- les apostilles et ajouts : les traiter en fin de transcription ? c-a-d ne pas les replacer à leur place ? quid lorsqu'il y a des signes de renvoi ?

**Comment by sardinecan** [comment]
au passage, les suscrits peuvent être transcrits avec le clavier virtuel dans eScriptorium a priori

**Comment by saranadal** [comment]
@sardinecan Je suis d'accord, dès le début je trouvais que ce que proposait Nouvelle France se rapprochait de nos objectifs. En intervenant le moins possible, on évite le bruit qui fausserait la transcription automatique. Je crois que c'est ce qu'indique Ariane Pinche.

**Comment by saranadal** [comment]
Concernant les ratures, cela va vraiment dépendre des mains. Par exemple chez Villain ou Berville, ça peut être facile, mais chez Quirot c'est quasi impossible.
Pour les ajouts, je trouve qu'il faudrait les replacer à leur place et ne pas les traiter en fin de transcription.

**Comment by emchateau** [comment]
Il ne me semble pas que nous devions entraîner un modèle pour lire le texte raturé (même si en soit cela pourrait être intéressant). Par contre, nous avons besoin de marquer les passages raturés pour ne pas entraîner fautivement la machine. Utiliser un `<del>`.

Je suis d’accord avec le fait que notre cas d’usage est assez proche de celui de Nouvelles France numérique qui a déjà réfléchi à ces questions. Mais partir de nos besoins avant de choisir me semble la bonne démarche.

Pour les mots agglutinés, il y a clairement une difficulté.
